### ISSUE TITLE
[FEATURE]: Patch meta information of `torch.nn.functional.softmax()`

### ISSUE THREAD

**Issue by Cypher30** [hpcaitech/ColossalAI#2629]
This is a part of issue #2628, we will patch meta information of `torch.nn.functional.softmax()`

**Comment by FrankLeeeee** [comment]
At least have a proper title and description.

**Comment by Cypher30** [comment]
> At least have a proper title and description.


This issue is tracked by main issue #2628, is that ok?
